src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md

```diff
@@ -112,7 +114,64 @@ runnable example.
 
 ## Enable OTLP Exporter for all signals
 
-Content coming soon.
+A cross-cutting extension was added in v1.8.0 to simplify registration of the
+OTLP exporter for all signals (logs, metrics, and tracing).
+
+> [!NOTE]
+> The cross cutting extension is currently only available when using the
+  `AddOpenTelemetry` extension in the
+  [OpenTelemetry.Extensions.Hosting](../OpenTelemetry.Extensions.Hosting/README.md)
+  package.
+
+```csharp
+appBuilder.Services.AddOpenTelemetry()
+    .UseOtlpExporter();
+```
+
+The `UseOtlpExporter` has the following behaviors:
+
+* Calling `UseOtlpExporter` automatically enables logging, metrics, and tracing.
+  Calls to `WithLogging`, `WithTracing`, and/or `WithLogging` are NOT required
```

Comment:
This sounds a little confusing. We start off saying calls to the "With" extension methods are required and then go on to say however only the telemetry enabled using "With" methods would be exported.

Maybe update this to something like:

> When calling `UseOtlpExporter` method, you don't need to separately call `AddOtlpExporter` method for `WithLogging`, `WithTracing`, and/or `WithMetrics`.

Reply:
This is kind of tricky. The docs we have that I could find seem to all show "Sdk.Create*" style which really doesn't help for this cross-cutting extension using the hosting style. I decided to show some real examples of how to do it. LMK what you think.

Reply:
`WithLogging` - This is not a known stable way of configuring logging today so I think we should not mention it. 

To keep things simple, what do you guys think about something like this.

```
Using UseOtlpExporter automatically enables the OTLP exporter for logs, metrics, and traces. 
For additional sdk configuration, like adding meters to the MeterProvider, 
incorporating sources to the TracerProvider, or 
configuring resources for all three types of signals, additional setup will required. 
For more information, refer to the details below. (Followed by what we have below)
```

Reply:
I removed the mention of the "With" calls and simplified it down to just...

> Calling `UseOtlpExporter` automatically enables logging, metrics, and tracing however only telemetry which has been enabled will be exported.